libs/langchain/langchain/document_loaders/sharepoint.py

```diff
@@ -53,6 +55,12 @@ def lazy_load(self) -> Iterator[Document]:
         if self.object_ids:
             for blob in self._load_from_object_ids(drive, self.object_ids):
                 yield from blob_parser.lazy_parse(blob)
+        if self.folder_id:
```

Comment:
is the logic here identical to the folder_path logic? if so maybe we just clarify in docstring that folder_path can be a path or an id?

Reply:
Apologies @baskaryan , I used the incorrect method. It should be the get_item() method when using ID, not the get_item_by_path() method. I have fixed this in the new commit.